src/lib/dialog/dialog-config.ts

```diff
@@ -32,6 +32,9 @@ export class MdDialogConfig {
   /** Position overrides. */
   position?: DialogPosition;
 
+  /** Classes to add to the dialog container. */
+  containerClass?: string | string[] = '';
```

Comment:
Do you need a class on the dialog container or on the root overlay element? I'd prefer to do the latter if we have to do this at all, as the dialog-container is meant to be a internal implementation detail